spec/spec_helper.rb

```diff
@@ -49,9 +49,11 @@ def has_internet?
 
   # We can't support outerWidth and outerHeight without a visible window.
   # We focus the next window instead of failing when closing windows.
-  # Node #send_keys is not yet implemented.
   c.filter_run_excluding :full_description => lambda { |description, metadata|
-    description =~ /Capybara::Session webkit Capybara::Window #(size|resize_to|maximize|close.*no_such_window_error|send_keys)/ || description =~ /Capybara::Session webkit node #send_keys/
+    not (description =~ /Capybara::Session webkit node #send_keys should send a string of keys to an element/) && (
```

Comment:
I had to take a minute to think about how the `not` would combine with `&&` here. What do you think about using `!~` here instead?